stretch-assignment/digital_timer/index.js

```diff
@@ -0,0 +1,84 @@
+class Timer{
+    constructor(){
+        this.tens=0
+        this.ones=0
+        this.sec=0
+        this.hs=0
+        this.bin=false
+        this.begin=false
+    }
+    reboot(){
+        this.tens=0
+        this.ones=0
+        this.sec=0
+        this.hs=0
+        this.bin=false
+        this.begin=false
+        this.read()
+    }
+    increment(){
+        if(this.tens===1 || this.begin==false){
+            return
+        }
+        this.hs++
+        if(this.hs>9){
+            this.hs=0
+            this.sec++
+            this.bin=true
+        }
+        if(this.sec>9){
+            this.ones++
+            this.sec=0
+        }

```

Comment:
I am curious is there a reason you didn't use `elseif` here?

Reply:
this is one of those times where I need to make sure that code an overflow. In fact, this can ONLY happen if the first one triggers. I'm using the overflow of the lower numbers to trigger overflow of the higher numbers.